docs/integrations/sources/typeform.md

```diff
@@ -4,44 +4,65 @@ This page guides you through the process of setting up the Typeform source conne
 
 ## Prerequisites
 
-* token - The Typeform API key token.
-* start\_date - Date to start fetching Responses stream data from.
-* form_ids (Optional) - List of Form Ids to sync. If not passed - sync all account`s forms.
+- [Typeform Account](https://www.typeform.com/)
+- Start Date - Date to start fetching Responses stream data from.
```

Comment:
They don't need this here, the pre-requisites section is for things they need before they even start setting up the connector
```suggestion
```